src/components/Tooltip/Tooltip.tsx

```diff
@@ -103,6 +105,15 @@ export const Tooltip: FC<TooltipProps> = React.memo(
         `${tooltipId?.current}-reference`
       );
 
+      const isMobile: boolean = isIOS() || isAndroid();
```

Comment:
so is the fix explicit to these mobile OSes? Is it possible to apply this fix when a device is generically touch enabled? Microsoft Surface comes to mind.

Reply:
I'll add a Surface method too that takes this into account, detecting based on touch only is not reliable.